org.freedesktop.Platform.Icontheme.EndlessOS.appdata.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="runtime">
+  <id>org.freedesktop.Platform.Icontheme.EndlessOS</id>
+  <metadata_license>CC0-1.0</metadata_license>
```

Comment:
Is this the correct license for our icons?

Reply:
Ohh that just the appdata file licence, I havent added the project licence field, but I should